src/views/dashboard/destination-details/destination-details.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useMemo, useState } from "react";
+import { useLocation, useParams } from "react-router-dom";
+import qs from "query-string";
+
+import { getAccomodationsForPlace } from "../../../services/api/destination/accomodationService";
+import Card from "../../../components/card/card";
+import Filter from "./filter/filter";
+
+import "./destination-details.scss";
+
+const DestinationDetails = () => {
+  const params = useParams();
+  const { search } = useLocation();
+  const [accomodations, setAccomodations] = useState<Array<any>>([]);
+  const queryParams = useMemo(() => qs.parse(search), [search]);
+
+  useEffect(() => {
+    fetchDestination();
+    fetchAccomodations();
+  }, [params]);
+
+  useEffect(() => {
+    fetchAccomodations();
+  }, [queryParams]);
+
+  const fetchDestination = async () => {
+    // @TODO
+  };
+
+  const fetchAccomodations = async () => {
+    let queryString = "";
+    if (Object.keys(queryParams).length > 0) {
```

Comment:
We should create helper function inside shared/utils/something.js... since this is common logic for building query string